language_bindings/rust/examples/iir_blur/src/main.rs

```diff
@@ -0,0 +1,80 @@
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+
+include!(concat!(env!("OUT_DIR"), "/iir_blur.rs"));
+
+use halide::runtime::{HalideBuffer};
+use halide::runtime::runtime_bindings::{halide_buffer_t, halide_type_t};//todo rust enum type to runtime mod and possibly escape from useing buffer_t
+use image::io::Reader;
+
+fn main(){
+
+    println!("halide mainish thing");
+
+    let img = Reader::open("images/cat.png")
```

Comment:
I can't comment on the image itself, so I'll comment here. Where is that image from? What is its license?